_docs/language-support.md

```diff
@@ -25,3 +25,27 @@ To help lay out what parts of Dockstore are available in which languages, we cov
 </sup>
 
 <sup> [4] All verified Dockstore WDL tools/workflows were tested successfully. However, we anticipate that more testing is needed for WDL workflows that use language features not contained within that dataset.</sup>
+
+
+## Converting File-path Based Imports to Public http(s) Based Imports
```

Comment:
In this context, we probably specify that this is for WDL. 
Also, watch out for broken links due to the move